src/index.js

```diff
@@ -53,7 +57,38 @@ export function extractAllAuthorsFromLibrary(library, flags = []) {
     }
   }
 
-  return addFlagsInResponse(useLevenshtein(authors), flags);
+  const authorsWithFlags = addFlagsInResponse(useLevenshtein(authors), opts.flags);
+
+  if (opts.domainInformations === true) {
+    return addDomainInformations(authorsWithFlags);
+  }
+
+  return authorsWithFlags;
```

Comment:
I would use a ternary operator here but it's up to you

```
return opts.domainInformations ? addDomainInformations(authorsWithFlags) :
    authorsWithFlags;
```

Reply:
I prefer to stay with the `if` version, since I don't know if I will add another opts later🤔